bot/src/main/java/com/community/tools/service/discord/RegisterCommand.java

```diff
@@ -84,6 +86,20 @@ public void run(SlashCommandEvent command) {
     userRepository.save(user);
   }
 
+  private User getUser(String userId) {
+    Optional<User> userOptional = userRepository.findByUserId(userId);
+    if (userOptional.isEmpty()) {
+      messageService.addRoleToUser(guildId, userId, newbieRoleName);
+      User user = new User();
+      user.setUserId(userId);
+      user.setGuildId(guildId);
+      user.setDateRegistration(LocalDate.now());
+      return user;
```

Comment:
Shouldn't we save the created user into db in such case?

Reply:
User is saved a bit later, when he passes the registration and submits his GitHub username. But if it is more convenient, I can add that line here

Reply:
If we save the data later, it's alright.